src/Tribe/Date_Utils.php

```diff
@@ -459,6 +459,30 @@ public static function is_all_day( $all_day_value ) {
 			);
 		}
 
+		public static function range_coincides( $range_1_start, $range_1_end, $range_2_start, $range_2_end ) {
```

Comment:
I was looking at it, couldn't we create something that allows more than 2 ranges? just Throwing the idea out in the :wind_chime: